src/chapters/loops/studio.rst

```diff
@@ -0,0 +1,63 @@
+Loop Studio
+============
+
+Now that we’ve launched our shuttle, let’s use loops (iteration) to
+automate some tasks.
+
+Part A (Put dinner together)
+----------------------------
+#. First, initialize variables to store the following arrays.  Remember to use descriptive names.
+    | Protein options: [‘chicken’, ‘pork’, ‘beef’, ‘fish’, ‘beans’, ‘tofu’]
+    | Grain options: [‘rice’, ‘pasta’, ‘potato’, ‘quinoa’, ‘crackers’, ‘corn’]
+    | Vegetable options: [‘peas’, ‘green beans’, ‘edamame’, ‘broccoli’,
+        ‘asparagus’, ‘kale’]
+    | Beverage options: [‘juice’, ‘milk’, ‘soy milk’, ‘soda’, ‘tea’, ‘water’]
+    | Dessert options: [‘apple’, ‘banana’, ‘ice cream’, ‘chocolate’, ‘kiwi’,
+        ‘more kale’]
+
+    Control+click (or right-click) to: `Code here <https://repl.it/@launchcode/LoopstudiopartsAandC>`__
+
+#. Construct a ``for`` loop that assembles a meal for each of 6 astronauts.
+    a. The meals must include one item from each of the source arrays.
```

Comment:
Are these items to be selected at random?

Reply:
The students have not yet learned how to use the Math module, so the meal assembly will just iterate sequentially through the arrays.  (That's why they are all the same length).